src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -778,6 +780,14 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 else
                         arg_enabled = r;
 
+        } else if (proc_cmdline_key_streq(key, "systemd.gpt_auto.relax_var_check")) {
+
+                r = value ? parse_boolean(value) : 0;
```

Comment:
I understand you follow the code for `systemd.gpt_auto`, but `arg_relax_var_check` is boolean. Please use `false`, instead of `0` here.

If you'd like, please replace `1` with `true` in the above.

Reply:
`r` here is a `int` variable which might contain error results, so `0` is appropriate here.

Reply:
Yeah, `r` is an integer, but it stores a negative errno or boolean. It does not store e.g. `100`. So, please please use `true/false` instead of `1/0`. That's our coding style.